source/docs/style-hooks.md

```diff
@@ -147,3 +147,47 @@ Here is a list of sywac styles out in the wild.
 > Tip:
 >
 > Know of a sywac style package, or have created your own? Submit a pull request for this documentation and add it to the list!
+
+## Using output settings in your style
+
+Your style hooks also have access to the user's [.outputSettings](/docs/sync-config.html#outputSettings-settings) configuration. Each property (such as `.indent`, `.lineSep`, etc.) is available as a property on `this`.
```

Comment:
Think I'd prefer the link here to point to just `/docs/sync-config.html#outputSettings` (the explicit anchor) instead, but we can clean that up later.